Student_BCOs/Tomas_Kanholm_BCO_000514.json

```diff
@@ -17,12 +33,95 @@
+                "orcid": "https://orcid.org/0000-0001-9705-2461",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Roberta Tarallo",
+                "orcid": "https://orcid.org/0000-0001-9668-3632",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Alessandro Weisz",
+                "orcid": "https://orcid.org/0000-0003-0455-2083",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
             }
         ]
     },
     "usability_domain": [
-        "Pipeline for hight-throughput analysis of small non-coding RNA data obtained by smallRNA-seq",
-        "Adapted from Giurato, Giorgio, Maria Rosaria De Filippo, Antonio Rinaldi, Adnan Hashim, Giovanni Nassa, Maria Ravo, Francesca Rizzo, Roberta Tarallo, and Alessandro Weisz. “IMir: An Integrated Pipeline for High-Throughput Analysis of Small Non-Coding RNA Data Obtained by SmallRNA-Seq.” BMC Bioinformatics 14 (December 13, 2013): 362. https://doi.org/10.1186/1471-2105-14-362."
+        "Pipeline for high-throughput analysis of small non-coding RNA data obtained by smallRNA-seq",
+        "Adapted from Giurato, Giorgio, Maria Rosaria De Filippo, Antonio Rinaldi, Adnan Hashim, Giovanni Nassa, Maria Ravo, Francesca Rizzo, Roberta Tarallo, and Alessandro Weisz. “IMir: An Integrated Pipeline for High-Throughput Analysis of Small Non-Coding RNA Data Obtained by SmallRNA-Seq.” BMC Bioinformatics 14 (December 13, 2013): 362. https://doi.org/10.1186/1471-2105-14-362.",
+        "This is single integrated pipeline software (iMIR v 1.0.0) for for high-throughput analysis of small non-coding RNA data obtained by smallRNA-seq. Description domain pipeline steps are named after the buttons a user would press to execute those steps.",
+        "Note: The software version for some execution domain software prerequisites were not listed in the iMir documentation. These include CutAdapt, SQUID, PDF::AP12, and Randfold.In cases where software version information was not available, the version number is listed as the latest version available as of the publication date of the paper (December 13, 2013)"
     ],
     "extension_domain": {},
     "description_domain": {
```

Comment:
Are these step names correct? See #117

Reply:
As per my [comment on issue #177](https://github.com/biocompute-objects/GW-SMHS-BIOC6223/issues/117#issuecomment-542297393), these step names are derived from the buttons a user would click to execute the required step in the integrated pipeline software my BCO is based on. Professor Keeney Ok'd this approach, so I think it should be ok? I also made a note to that effect in the usability domain to help make it clear.

If you feel strongly about this please let me know how you think I should go about it!

Thanks!

Reply:
Ahh yes. Thanks for reminding me @tkanholm ! No, this is fine.